templates/CRM/AdditionalPayment/Form/Payment/Refund.tpl

```diff
@@ -0,0 +1,20 @@
+{literal}
+    <script type="text/javascript">
+         jQuery( document ).ready(function() {
```

Comment:
There are few comments here

1. Please make sure you use ES6
2. Make sure you use `CRM.$` to use jQuery version in the CiviCRM.  We should not rely on CMS and to avoid any conflicts if they don't have the same version or not all CMS adding jQuery to all the page  - Please see go through the reference here https://docs.civicrm.org/dev/en/latest/standards/javascript/#javascript-reference
3. Make sure your code has a consistency and following CiviCRM PHP's coding [standard](https://docs.civicrm.org/dev/en/latest/standards/javascript/#coding-standards) since we don't have the automated Jslinter i.e. ` if(jQuery(this).attr('class')` vs if (text == "Submit Credit Card Refund"), it could be worth running CiviLint against and updated JS/New file.